added favicon

CodingSpecies opened the pull request:
I updated the public folder files to change it to the EddieHub branding. 

It is this logo:
<img src="https://user-images.githubusercontent.com/624760/114314271-ea156a80-9af1-11eb-97ca-977be7565aa6.png" height=80 />

#218

aayush89890 reviewed (approved):
Looks good, Thanks @CodingSpecies for you PR.

eddiejaoude commented:
Thank you, I think the favicon is the wrong size though? It should be about 128x128 pixels and usually under 1k (IIRC)

CodingSpecies commented:
> 
> 
> Thank you, I think the favicon is the wrong size though? It should be about 128x128 pixels and usually under 1k (IIRC)

I agree, I'll change the sizes

CodingSpecies commented:
I am not sure how to alter the image sizes, I am trying, but not really sure.
thank you.

Panquesito7 commented:
> Thank you, I think the favicon is the wrong size though? It should be about 128x128 pixels and usually under 1k (IIRC)

AFAIK, it should be 32x32. With that size it won't work, I think.

eddiejaoude commented:
I would use an online Favicon converter

CodingSpecies commented:
this is 32x32:


![logo512_c0R_icon](https://user-images.githubusercontent.com/70807500/134817793-41e0ac1e-29a1-4e53-aea9-2477430721e8.png)

Panquesito7 commented:
> this is 32x32:
> 
> ![logo512_c0R_icon](https://user-images.githubusercontent.com/70807500/134817793-41e0ac1e-29a1-4e53-aea9-2477430721e8.png)

I think that looks great. Try to use that and let us know how it looks. 🙂

eddiejaoude commented:
Looks much better 👍

Panquesito7 commented:
I'm not sure if it works by just adding the favicon. I guess we need to modify some code. 🤷‍♂️

CodingSpecies commented:
In the code the favicon is used by its file name, so I just rewrote the file, and changed the images, but the files are still called the same, so I think the code is ok, but I am not sure.

CodingSpecies commented:
@eddiejaoude @Panquesito7 completed and changed the favicon and icons sizing. Thank you.

Panquesito7 reviewed (approved):
Cool, thanks! 🚀

CodingSpecies commented:
Thank you @eddiejaoude @Panquesito7
